### DISCUSSION TITLE
Direct Code Editor Missing

### DISCUSSION THREAD

**Discussion by Musaddaq-Nawaz** [Sketchware-Pro/Sketchware-Pro#726] [Q&A] [answered]
Please add direct code editor

**Comment by rukamori** [comment, marked as answer]
https://discord.com/channels/790686719753846785/790749749522464779/908817727878201424

**Reply by j-px** [reply]
Or here a screenshot of the message:  
![image](https://user-images.githubusercontent.com/40118124/215293121-884e664d-769b-479b-9d55-c807af889798.png)
